platform-service/src/unstract/platform_service/controller/platform.py

```diff
@@ -26,6 +26,7 @@
     password=Env.PG_BE_PASSWORD,
     host=Env.PG_BE_HOST,
     port=Env.PG_BE_PORT,
+    options=f"-c application_name={Env.APPLICATION_NAME}",
```

Comment:
@Deepak-Kesavan what does this -c do?

Reply:
@chandrasekharan-zipstack Check this for more details.

https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNECT-OPTIONS
https://www.postgresql.org/docs/current/config-setting.html#CONFIG-SETTING-SHELL

From my understanding, -c is a commanline arg.